### ISSUE TITLE
"Safari Busy" when VoiceOver Enabled

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#267]
I thought it was system version (Mac OS 10.13.6) and it might still be partly due to my system version, but for some time now, I have been having trouble using RaP with VoiceOver Enabled.

Until last week, I think I could still get the sim to respond by restarting VO, but today, I am constantly getting "Safari Busy" when tabbing through the sim.

So, today, I also tested out some other sims (MAL and RIAW). I could use those sims, and they performed as expected without producing "Safari Busy".

I did notice there were context response pile-ups for RIAW, though. I'll make a note in the RIAW repo in case it's been affected by any recent changes to the utterance queue.

**Comment by terracoda** [comment]
So my system is Mac OS 10.13.6 and Safari is Version 13.1.2 (13609.3.5.1.5)

**Comment by terracoda** [comment]
I do plan on upgrading very soon, but still thought it was important to post this issue in case it is not directly related to my system.

**Comment by terracoda** [comment]
Basically, I mainly hear, "Safari Busy" when I tab through the sim.

**Comment by zepumph** [comment]
@KatieWoe, could you have a QA person do a very limited test on your latest mac/ VO with master of Ratio and Proportion to see if this is an issue generally, or from @terracoda's system/version?

**Comment by terracoda** [comment]
Oh, this does not happen when I uncheck the "debugger" checkbox in PhET Bookmarks.

**Comment by terracoda** [comment]
I hadn't noticed until recently that the debugger is ticked by default.

**Comment by terracoda** [comment]
@KatieWoe, do you want to verify that this is due to the debugger, or should I just close the issue?

**Comment by KatieWoe** [comment]
I think that we can leave it there if you are fairly confident. It may be a good thing to check when the sim next goes for testing though.

**Comment by terracoda** [comment]
@KatieWoe, I am no longer confident.
It seems like it was a fluke. 
The sim is now working with the debugger on and with the debugger is off.

Good news is... I can use the sim with VO now. 
Not sure why it doesn't work sometimes.

**Comment by KatieWoe** [comment]
Could it be related to https://github.com/phetsims/molecules-and-light/issues/349?

**Comment by zepumph** [comment]
Hmm That is a good point, from that issue I took that this "safari busy" normally has to do with performance. I'll take a look to see if there are any obvious performance problems that could be causing this in the sim-specific code.

**Comment by terracoda** [comment]
Good news, I am not getting "Safari Busy" today.

**Comment by zepumph** [comment]
I have not been able to work on this because I don't have access to a mac. I think I would prefer to not have this block RC, and then to be able to have QA confirm that this is a problem consistently enough, and we can fix it while in RC. @terracoda does that seem like an appropriate plan, or should I do further investigation before RC?

**Comment by terracoda** [comment]
@zepumph, I don't think this issue should block RC. I have not heard Safari busy in a long time, so perhaps performance has been improved through other work.

**Comment by zepumph** [comment]
Ok sounds good. I'll mark it for confirmation by QA during RC. Thanks!

**Comment by muedli** [comment]
I didn't see you this come up in https://github.com/phetsims/QA/issues/601. I did my macOS Voice Over testing on macOS 10.15.7 with Safari 14.0.2.

**Comment by muedli** [comment]
I'm going to close this issue, @zepumph.
